hr_employee_passport/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Hr Employee Passport',
+    'summary': """
+        Allow to link to store employee's passports""",
```

Comment:
I'd use or link or store not both, but it's not big deal.

Reply:
Ups. I corrected it.